apps/arweave/src/ar_config.erl

```diff
@@ -190,6 +190,17 @@ parse_options([{<<"verify_samples">>, <<"all">>} | Rest], Config) ->
 parse_options([{<<"verify_samples">>, Opt} | _], _) ->
 	{error, {bad_type, verify_samples, number}, Opt};
 
+parse_options([{<<"vdf">>, Mode} | Rest], Config) ->
+	ParsedMode = case Mode of
+		"openssl" ->openssl;
```

Comment:
should we also include `opensslite` for consistency with the CLI args?

Reply:
As discussed, we probably want just drop openssl, and just have one option "openssl" which just will use openssllite